source/Halibut.Tests/Support/LatestClientAndLatestServiceBuilder.cs

```diff
@@ -138,6 +139,18 @@ public LatestClientAndLatestServiceBuilder WithServiceStreamFactory(IStreamFacto
             return this;
         }
 
+        public LatestClientAndLatestServiceBuilder WithServiceConnectionsObserver(IConnectionsObserver countingServiceObserver)
```

Comment:
nit: `countingServiceObserver`

Reply:
I didn't see these nits (juggling too many PRs...).

But here is a [follow up PR](https://github.com/OctopusDeploy/Halibut/pull/493) to address them